anchore_engine/analyzers/manager.py

```diff
@@ -29,8 +29,8 @@ def apply_hints(analyzer_report, unpackdir):
     # the config does not explicitly enable hints processing, effectively disabling
     # hints processing.
     for engine_entry in utils.content_hints(unpackdir=unpackdir):
-        pkg_type = engine_entry.get("type")
-        if pkg_type:
+        pkg_type = engine_entry.get("type", "")
+        if pkg_type and pkg_type != "go":
             handler = syft.modules_by_engine_type[pkg_type]
```

Comment:
This was failing with KeyError when pkg_type == "go", causing all go hints to fail.